report_qweb_encrypt/i18n/es.po

```diff
@@ -0,0 +1,81 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# 	* report_qweb_encrypt
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 14.0\n"
+"Report-Msgid-Bugs-To: \n"
+"Last-Translator: Automatically generated\n"
```

Comment:
Don't touch translation files.

You may regenerate pot file if you would like to. Both pot and po will be regenerated by the bot anyway.

Reply:
Fixed, thanks!